### DISCUSSION TITLE
How to convert word to HTML

### DISCUSSION THREAD

**Discussion by Addimlal** [plutext/docx4j#537] [Q&A] [answered]
Hi,

I want to convert DOCX document to HTML page and would like to know how if it's that possible, if yes how I can do it using this library.

It will be really helpful to have getting start for DOCX to HTML.

Thank you

**Comment by plutext** [comment, marked as answer]
Yes, see sample https://github.com/plutext/docx4j/blob/VERSION_11_4_8/docx4j-samples-docx4j/src/main/java/org/docx4j/samples/ConvertOutHtml.java
